### PR TITLE
Create ESPN.PY

### PR DESCRIPTION
**Pull request by Anshulsatle**
please check it is good or not

### CODE REVIEW

**Review by santoshilondhe — APPROVED** [review]
LGTM

**Review by berdedisha — APPROVED** [review]
LGTM!

File: Anshul/ESPN.PY [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import time
+
```
Comment (pcube0608) [inline Anshul/ESPN.PY]: Please remove extra lines from here

File: Anshul/ESPN.PY [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import unittest
+
+from selenium.webdriver import ActionChains
+from selenium.webdriver.common.by import By
+
+
+class ESPN(unittest.TestCase):
+    def setUp(self) -> None:
+        global IPL1
```
Comment (pcube0608) [inline Anshul/ESPN.PY:10]: Can we rename this object as driver ?

File: Anshul/ESPN.PY [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        IPL1.maximize_window()
+
+
+    def test_OutPlyerNmae(self):
+        KeySeriessexction=IPL1.find_element(By.XPATH,"//span[text()='World Cup Super League']")
+        Ac=ActionChains(IPL1)
+        Ac.move_to_element(KeySeriessexction).perform()
+        IPL2023=IPL1.find_element(By.XPATH,"//span[text()='IPL 2023']/parent::a")
+        IPL2023.click()
+        time.sleep(20)
```
Comment (pcube0608) [inline Anshul/ESPN.PY:22]: Can we replace this hardcoded wait with explicit wait ?
Have we tried applying explicit wait ?
Please check if explicit wait works here.

File: Anshul/ESPN.PY [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        Ac=ActionChains(IPL1)
+        Ac.move_to_element(Result).click().perform()
+        Lastmatchwin=IPL1.find_element(By.XPATH,"//div[@class='ds-block']//a")
+        Lastmatchwin.click()
+        time.sleep(13)
+        #IPLMenu=IPL1.find_element(By.XPATH,"//span[text() = 'Indian Premier League 2023']/../ i")
+        #IPLMenu.click()
+        # ipl2021=IPL1.find_element(By.XPATH,"//li[@title='Indian Premier League 2023']")
+        # Ac1=ActionChains(IPL1)
+        # Ac1.move_to_element(ipl2021).click().perform()
```
Comment (pcube0608) [inline Anshul/ESPN.PY:36]: Please remove commented code if not required

File: Anshul/ESPN.PY [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        lastIpl=IPL1.find_element(By.XPATH,"//div[@class='ds-w-[288px] card scorecard']")
```
Comment (pcube0608) [inline Anshul/ESPN.PY:37]: The class name contains white spaces and looks volatile.
Can we use contains in xpath to make it more generic

**Review by pcube0608 — CHANGES_REQUESTED** [review]

File: Anshul/ESPN.PY (hunk 2 above)
Comment (pcube0608) [reply Anshul/ESPN.PY:10]: Done

**Review by pcube0608 — APPROVED** [review]
All changes are done. Thanks @Anshulsatle

**Pull request merged**